### ISSUE TITLE
CCD image's issue with Kronos 2.1.2

### ISSUE THREAD

**Issue by LuismaSP89** [FCare/Kronos#864]
Guardian heroes got a black screen when you try to start/load a game, I only can reach the main menu "After the intro" after this you can't start a new game or load a previous game, only blackscreen with no sound. This game worked in previous versions of Kronos.

What's the full name of the game (including region) ?
Guardian Heroes (PAL)

What's your frontend ? (standalone, libretro, ...) ?
Standalone

What's your OS ? (windows, linux, OSX, ...) ?
Windows 10 x64

What's your hardware ? (CPU, GPU, ...) ?

i7 4790k
16 GB DDR3
GTX 970

What's the storage type of the game (official cd, burned cd or cd image) ?
cd image

If applicable, what's the format of your cd image ? (cue/bin, ccd/img, iso, chd, ...)
ccd+img+sub

Are you using any setting different from default ? If so, precise
Tested with default and custom settings, no difference

Did this game work previously ? Do you remember when ? Is it happening with other games ?
Yes, worked in previous versions of Kronos. At this moment is the only game I got with this issue.

If applicable, provide error messages, logs, screenshots, or whatever additional information that could help debugging this

**Comment by fafling** [comment]
With the US CD or PAL redump, I don't reproduce that problem with public v2.1.2 64 bit.

**Comment by barbudreadmon** [comment]
@LuismaSP89 i was unable to reproduce this either, are you using WIP ?

**Comment by LuismaSP89** [comment]
@barbudreadmon No, I'm using the 32 bit version from emufrance.

**Comment by FCare** [comment]
the 64 bits may have better performance since you are using win10 64 bits...

**Comment by LuismaSP89** [comment]
@FCare Ok, thank you, I'll try the 64 bit version. I'm using x32 version because in the past, x64 version had a worse performance at least for me, with some bugs and errors that x32 doesn't have.

**Comment by LuismaSP89** [comment]
Nope, the issue persist. Tested with x32 and x64 builds, all default settings/clean install.

**Comment by barbudreadmon** [comment]
can you try with a redump version of the game, to rule out an issue with ccd dumps ?

**Comment by barbudreadmon** [comment]
Actually got my hands on a ccd dump, so i can confirm : i get a black screen with it

**Comment by LuismaSP89** [comment]
I see... ok, I'm going to edit the issue's title to "CCD image's issue with Kronos 2.1.2"
